server/events/command_runner.go

```diff
@@ -254,7 +254,7 @@ func (c *DefaultCommandRunner) RunPRReviewCommand(ctx context.Context, repo mode
 		Pull:              pull,
 		PullStatus:        status,
 		HeadRepo:          repo,
-		Trigger:           command.AutoTrigger,
```

Comment:
For some context, i initially made this field to allow us to differentiate between things that are run automatically (ie. part of a pull update event) or things that are run manually (such as a comment from the user). 

We are changing the definition of this a little bit by using it in this way since now it's starting to look like a source? That might be fine i guess, but just letting you know